packages/components/src/Label/Label.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { LabelHTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./Label.module.scss"
+type LabelType = "text" | "checkbox" | "toggle" | "radio"
+
+export type LabelProps = OverrideClassName<
+  LabelHTMLAttributes<HTMLLabelElement>
+> & {
+  children?: React.ReactNode
+  labelText?: string | React.ReactNode
```

Comment:
This is honestly extremely confusing API because both these props do the same thing except when they are next to something like a checkbox (which still, technically could've just added classes and remained using `children` - less you'd have text to both the left and the right of the checkbox at the same time).

Give it some thought on whether you want to clean it up a little or not, given now is safer to do so as opposed to after teams start to use it (that being said, this component is too complex for what it needs to be anyway, so you can choose to just say it gets simplified in a rewrite).

Reply:
Not on the fly 😅